qa/paqa_devs.c

```diff
@@ -1,13 +1,13 @@
 /** @file paqa_devs.c
-	@ingroup qa_src
+    @ingroup qa_src
     @brief Self Testing Quality Assurance app for PortAudio
- 	Try to open each device and run through all the
- 	possible configurations. This test does not verify
+    Try to open the default devices and run through all the
+    possible configurations. This test does not verify
```

Comment:
If you are going to update this comment, I recommend that you say what it actually does. e.g.:

```
Try to open devices and run through all possible configurations. By default, open only the default devices. Command line options support opening every device, or all input devices, or all output devices.
```